### DISCUSSION TITLE
Is it possible to interact with VMs inside the cluster ?

### DISCUSSION THREAD

**Discussion by Sensei-CHO** [CRASH-Tech/proxmox-operator#10] [Q&A]
Hi ! This project seems really great for homelabers !
I'm curious about what this CRDs can do.
I recently tried KubeVirt which is really helpful if you have to run VMs or workloads which cannot work inside containers.
Is proxmox-operator working like this ? Can I run worklaods on the proxmox VMs ?

**Comment by CRASH-Tech** [comment]
Hello! It's can create VMs, delete(if crd deleted and protection off) and modify VMs.
It works with proxmox api and sync VMs with crd resources.
Of cource you can run workloads on this VMs.
You can use already prepared VM template or use cloud-init for provision
